src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -85,7 +96,57 @@ public async Task RunApplicationAsync(CancellationToken cancellationToken = defa
         {
             AspireEventSource.Instance.DcpModelCreationStop();
         }
+    }
+
+    private async Task StartDashboardAsync(CancellationToken cancellationToken = default)
+    {
+        if (_options.Value.DashboardPath is not { } dashboardPath)
+        {
+            throw new DistributedApplicationException("Dashboard path empty or file does not exist.");
+        }
+
+        var fullyQualifiedDashboardPath = Path.GetFullPath(dashboardPath);
+        var dashboardWorkingDirectory = Path.GetDirectoryName(fullyQualifiedDashboardPath);
+
+        var dashboardExecutableSpec = new ExecutableSpec();
+        dashboardExecutableSpec.ExecutionType = ExecutionType.Process;
+   
+        dashboardExecutableSpec.ExecutablePath = "dotnet";
+        dashboardExecutableSpec.Args = [fullyQualifiedDashboardPath];
+        dashboardExecutableSpec.WorkingDirectory = dashboardWorkingDirectory;
+
+        var grpcEndpointUrl = _dashboardServiceHost.DashboardServiceUri?.ToString();
+        var otlpEndpointUrl = Environment.GetEnvironmentVariable("DOTNET_DASHBOARD_OTLP_ENDPOINT_URL");
+        var dashboardUrl = Environment.GetEnvironmentVariable("ASPNETCORE_URLS");
+
+        var environment = new List<EnvVar>();
+        environment.Add(new EnvVar()
+        {
+            Name = "DOTNET_DASHBOARD_GRPC_ENDPOINT_URL",
+            Value = grpcEndpointUrl
+        });
+        environment.Add(new EnvVar()
+        {
+            Name = "ASPNETCORE_URLS",
+            Value = dashboardUrl
+        });
+        environment.Add(new EnvVar()
+        {
+            Name = "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL",
+            Value = otlpEndpointUrl
+        });
+        environment.Add(new EnvVar()
+        {
+            Name = "ASPNETCORE_ENVIRONMENT",
+            Value = "Development"
+        });
+
+        dashboardExecutableSpec.Env = environment;
+
+        var dashboardExecutable = new Executable(dashboardExecutableSpec);
+        dashboardExecutable.Metadata.Name = "aspire-dashboard";
```

Comment:
What happens if someone has a resource called `aspire-dashboard`?

Reply:
We won't launch the dashboard from the workload.